contracts/truefi2/TrueFiPool2.sol

```diff
@@ -662,6 +662,19 @@ contract TrueFiPool2 is ITrueFiPool2, IPauseableContract, ERC20, UpgradeableClai
         return liquidValue().mul(BASIS_PRECISION).div(_poolValue);
     }
 
+    /**
+     * @dev Ratio of liquid assets in the pool after lending
+     * @param amount Amount of asset being lent
+     * @return Calculated ratio in basis points
+     */
+    function proFormaLiquidRatio(uint256 amount) external override view returns (uint256) {
```

Comment:
I'll add tasks to delete the original `liquidRatio()` and `utilization()` views to avoid cluttering TFP2 with unused functions. (No action needed in this PR.)

Reply:
We can always use this method if needed and just pass 0 as the amount if needed. That's a good idea.